### ISSUE TITLE
Gap undefined in Time Series Tutorial

### ISSUE THREAD

**Issue by theaiacademy** [h2oai/tutorials#73]
The Driverless AI Time Series Tutorial presented here: https://h2oai.github.io/tutorials/time-series-recipe-tutorial-retail-sales-forecasting/#3 does not include a step to define the "gap" between the training dataset and the test dataset:

- GAP BETWEEN TRAIN/TEST = 0 (weeks)

Looking at the datasets provided for this tutorial looks like there is 1 week gap:

- MAX(DATE.walmart_tts_small_train.csv) = 2012-04-27
- MIN(DATE.walmart_tts_small_test.csv) = 2012-05-04

shouldn't the GAP BETWEEN TRAIN/TEST set to "1" ?

**Comment by Castro38** [comment]
Hi and thank you for your question. Someone from our team will take a look at your questions and will provide an answer in a prompt manner. 

Regards, 
The H2O.ai Tutorials Team
